theme-list/types/themeList.ts

```diff
@@ -0,0 +1,5 @@
+import { ConfigType } from '@stitches/react/types/config';
+
+export type ThemeList = {
+  [key: string]: ConfigType.Theme;
```

Comment:
would be able to use the ThemeBase type as the restricted key for this type? Not sure if that's overkill.

Reply:
I tried and it bitched at me.